js/views/setupview.js

```diff
@@ -87,7 +87,7 @@ define(function(require) {
 			this._config = config;
 			this.render();
 
-			return Radio.account.request('create', config).then(function() {
+			return Radio.account.request('create', {config: config}).then(function() {
```

Comment:
in `accountservice.js`, you have `data: {config: config},`. Doesn't that mean you wrap that in a object property twice? Something like `{ config: { config: { ... } } }`?

Reply:
It was intended for easier use in the controller. My preference really... Will revert change.